wdio.conf.js

```diff
@@ -18,11 +18,14 @@ exports.config = {
     //
     specs: [
         './test/smoke/storyPage.js',
```

Comment:
Remove the list of the files

Reply:
should I leave only './test/smoke/storyPage.js', ?